pyproject.toml

```diff
@@ -9,6 +9,7 @@ authors = ["Christian Decker <cdecker@blockstream.com>"]
 python = "^3.8"
 pyln-client = { path = "contrib/pyln-client", develop = true }
 pyln-proto = { path = "contrib/pyln-proto", develop = true }
+pyln-grpc-proto = { path = "contrib/pyln-grpc-proto", develop = true }
```

Comment:
Strange that this is required.


it should be imported with pyln-testing

```
contrib/pyln-testing/pyproject.toml:27:pyln-grpc-proto = { version = "^0.1", optional = true }
contrib/pyln-testing/pyproject.toml:31:pyln-grpc-proto = { path = "../pyln-grpc-proto", develop = true, optional = true}
contrib/pyln-testing/pyproject.toml:34:grpc = ["pyln-grpc-proto", "grpcio"]
```

Reply:
The semantics of the `develop`-flag are confusing.

The `pyproject.toml`-file at our package-root will install `pyln-testing` with the `develop = true`.
By consequence, `pyln-testing` will be installed as editable. Any change in source files will take immediate effect.
However, it will not affect what sub-dependencies are installed. 

I remember filing an [issue](https://github.com/python-poetry/poetry/issues/9124) in the poetry repo about this behavior and was asked to close it immediately.

Reply:
Good to know, than it should definitely be added!